tests/test_generate_ics.py

```
@@ -425,6 +449,46 @@ def test_filename_subdivision(self):
+            self.assertFalse((temp_dir / "US_2025-personal.ics").exists())
+
+    def test_filename_output_and_suffix_are_mutually_exclusive(self):
+        with (
+            self.argv("US", "--output", "calendar.ics", "--output-suffix", "-personal.ics"),
+            self.assertRaises(SystemExit),
+        ):
+            IcsGenerator()
+
+    def test_output_suffix_does_not_consume_long_options(self):
+        with self.argv("US", "--output-suffix", "--list-languages"):
+            with self.assertRaises(SystemExit):
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)